vault/core.go

```diff
@@ -757,6 +760,9 @@ type CoreConfig struct {
 	// Use the deadlocks library to detect deadlocks
 	DetectDeadlocks string
 
+	// Disables tracking a leases role
```

Comment:
```suggestion
	// If no role-based lease-count quota is enabled, don't track lease counts by role
```

Reply:
I changed this comment to read: If no role-based lease count quota or role based rate limiting quota is enabled, don't track lease counts by role

Reply:
Sounds good! Thanks :)